components/bl-snackbar-component/src/components/icons/close.js

```diff
@@ -0,0 +1,13 @@
+export const Close = () => {
```

Comment:
file name: close, as it under icons directory
component name: CloseIcon - because we import it somewhere.